frontend/src/utils.js

```diff
@@ -6,8 +6,15 @@ export const formatDate = (dateString) => {
     return date.toLocaleDateString("en-US", options);
 };
 
-export const capitalize = (word) => {
-    return word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
+export const capitalize = (sentence) => {
+
+    const words = sentence.split(" ");
+
+    const capitalizedWords = words.map((word) => {
+        return word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
+    });
+
+    return capitalizedWords.join(" ");
```

Comment:
Good job writing this function. 

One thing to notice is that every function you are using here is non mutating, and returns a new object, so they can be easily chained:

```
return sentence.split(" ")
  .words.map((word) => {
    return word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
  })
  .join(" ");
```

It's the same thing, but without capturing intermediate results in variables. In many cases you don't need variables. I want to know your thoughts. What option seems better to you, and why?

Reply:
I like how the first one looks better and it's more organized to me and easier to read.